core/templates/pages/exploration-editor-page/services/exploration-objective.service.spec.ts

```diff
@@ -16,31 +16,38 @@
  * @fileoverview Unit tests for the ExplorationObjectiveService.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { importAllAngularServices } from 'tests/unit-test-utils.ajs';
-// ^^^ This block is to be removed.
+import { TestBed } from '@angular/core/testing';
+import { ExplorationObjectiveService } from './exploration-objective.service';
+import { ExplorationPropertyService } from './exploration-property.service';
+import { ExplorationRightsService } from './exploration-rights.service';
+import { ValidatorsService } from 'services/validators.service';
+import { NormalizeWhitespacePipe } from 'filters/string-utility-filters/normalize-whitespace.pipe';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
 
-require(
-  'pages/exploration-editor-page/services/exploration-objective.service.ts');
+describe('Exploration Objective Service', () => {
+  let service: ExplorationObjectiveService;
```

Comment:
Can you use a more specific name for this variable like `explorationObjectiveService` or `eos`?

Reply:
Done